extensions/spotify-beta/package.json

```
@@ -63,6 +63,24 @@
           "type": "textfield",
           "default": "20",
           "required": false
+        },
+        {
+          "name": "iconType",
+          "title": "Menu Bar Icon type",
```

Since these preferences are already nested in the Menu Bar command, I don't think we need to include in the title

I'd move this to be the first preference as well

```suggestion
          "title": "Icon type",
```